docs/modules/ROOT/nav.adoc

```diff
@@ -1,4 +1,5 @@
 * xref:index.adoc[Overview]
+* xref:upgrade-safe.adoc[Upgrade Safety]
```

Comment:
We could change the navigation order to have all the general guides be at the top.
We could even have the pages be sub pages of Overview.
* Overview
  * Extending Contracts
  * Upgrade Safety
  * Releases & Stability